internal/web/components/pkg_detail_btns/controller.go

```diff
@@ -17,20 +17,22 @@ type pkgDetailBtnsInput struct {
 	Status          *client.PackageStatus
 	Manifest        *v1alpha1.PackageManifest
 	UpdateAvailable bool
+	AutoUpdate      string
```

Comment:
I don't really see what the auto update has to do with the `pkg-detail-btns` component – it is not rendered in there. You only really show it on the `package.html` template. So you can remove your changes in this controller.

Reply:
If i remove this part from the code then the package details page doesnt even render also in the terminal i get errors stating 

```
Unexpected error rendering package-detail (argo-cd): template: package.html:31:50: executing "body" at <ForPkgDetailBtns>: wrong number of args for ForPkgDetailBtns: want 5 got 6
 – This is most likely a BUG – Please report it here: https://github.com/glasskube/glasskube
```

otherwise it's fine

Reply:
In package.html line 31 the `ForPkgDetailBtns` function is called. You need to make sure to also remove this additional parameter there.

Please note that only templates are "hot-reloaded" during development, not the changes in the go files. So if you change something in go code, please make sure that you restart the server.